### ISSUE TITLE
Change gradient descent step calculation in StudentAdmissions exercise

### ISSUE THREAD

**Issue by adrielvieira** [udacity/deep-learning-v2-pytorch#208]
The exercise in [StudentAdmissions notebook](https://github.com/udacity/deep-learning-v2-pytorch/blob/master/intro-neural-networks/student-admissions/StudentAdmissions.ipynb
) uses Log-loss as the error function, but it looks like it is updating the weights as if the error function was the SSE.

This exercise appears in the course before the SSE has been introduced, therefore the error function shouldn't change. Instead, I believe that the weight updates should not be using the error_term calculation.

**Comment by abhiojha8** [comment]
This is resolved by PR #250
